modules/history.js

```diff
@@ -198,7 +198,7 @@ History.prototype.validateEntryField = function(field, value) {
 History.prototype.set = function(fieldNameOrHistoryObject, historyValue) {
   // Is the first argument a field name or a full history object?
   if (typeof fieldNameOrHistoryObject === 'string') {
-    if (this.validateEntry(fieldNameOrHistoryObject, historyValue)) {
+    if (this.validateEntry(historyValue)) {
```

Comment:
Do we not need to ensure that the given value is valid for the specific field given in the first argument?